_15thnight/api/account.py

```diff
@@ -1,27 +1,24 @@
 from datetime import datetime, timedelta
-from flask import Blueprint, request, render_template, url_for
+from flask import Blueprint, current_app, request
 from flask.ext.login import (
     login_user, logout_user, login_required, current_user
 )
-from flask_mail import Message
 
+from _15thnight.core import (
+    send_password_reset, send_help_message, send_confirm_password_reset
+)
 from _15thnight.forms import (
     LoginForm, ChangePasswordForm, UpdateProfileForm, ResetPasswordForm,
     ForgotPasswordForm, csrf_protect
 )
 from _15thnight.models import Service, User
-from _15thnight.queue import queue_send_email
-from _15thnight.util import jsonify, api_error
-
-try:
-    from config import RESET_TOKEN_LIFE
-except:
-    from config import RESET_TOKEN_LIFE
+from _15thnight.util import api_error, jsonify, required_access
 
 
 account_api = Blueprint('account_api', __name__)
 
-reset_token_life = timedelta(hours=RESET_TOKEN_LIFE)
+reset_token_life = timedelta(
```

Comment:
This causes a runtime error

define this in reset_password. it is the only method that uses it and is always called within an application context.